### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by shr-project**

### CODE REVIEW

File: recipes-extended/qhull/qhull/0001-testqset-fix-build-with-conf-distro-include-security.patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Date: Mon, 9 Apr 2018 21:45:24 +0000
+Subject: [PATCH] testqset: fix build with
+ conf/distro/include/security_flags.inc
+
+* fixes:
+  http://errors.yoctoproject.org/Errors/Details/174716/
+  qhull/2012.1-r0/qhull-2012.1/src/testqset/testqset.c:217:9: error: format not a string literal and no format arguments [-Werror=format-security]
+         printf(promptstr);
+         ^~~~~~
+
```
Comment (bulwahn) [inline recipes-extended/qhull/qhull/0001-testqset-fix-build-with-conf-distro-include-security.patch:12]: Here, an Upstream-Status is missing.

**Comment by bulwahn** [comment]
Okay, pull request looks good. Let's wait till the CI is back and then merge it.

**Comment by shr-project** [comment]
Upstream-Status added.

**Comment by rojkov** [comment]
Looks good to me. Merging...

**Pull request merged**